demos/python_demos/llm_text_generation/servable_stream/model.py

```diff
@@ -170,13 +170,16 @@ def execute(self, inputs: list):
             generate_kwargs["stopping_criteria"] = StoppingCriteriaList(stop_tokens)
 
         ov_model_exec = self.ov_model.clone()
+        self.token_count = 0
         def generate():
-            ov_model_exec.generate(**tokens, **generate_kwargs)
+            result = ov_model_exec.generate(**tokens, **generate_kwargs)
```

Comment:
What's the reason we do not include special tokens? What are those special tokens?

Reply:
In this case special tokens are used for left/right padding. They are stripped to assure accurate token counting when batch_size > 1.